### ISSUE TITLE
Use Ember.run.debounce instead of custom debounce

### ISSUE THREAD

**Issue by mitchlloyd** [mitchlloyd/ember-autosave#6]
http://emberjs.com/api/classes/Ember.run.html#method_debounce

**Comment by mitchlloyd** [comment]
Thank. I'm in the process of converting this to an add on, and I'll probably take a peek at your code to implement this.

**Comment by Genkilabs** [comment]
After a lot of dev, I have started working with https://github.com/coalescejs/coalesce
It's an evolution of Ember Persistence Foundation EPF. There is a coalesce-ember binding which works with CLI https://github.com/coalescejs/coalesce-ember
It's not about autosaving specifically, but is more of a full data layer replacement. Take a look if you are curious.

**Comment by mitchlloyd** [comment]
Fixed by 3b25625e001f59ffd88cf17b12d94ee7cc192fb2

@Genkilabs Let me know if the new configuration is flexible enough to accommodate coalesce-ember.

```javascript
import Ember from 'ember';
import { AutosaveProxy } from 'ember-autosave';

export function initialize() {
  AutosaveProxy.config({
    saveDelay: 3000, // Wait 3 seconds after input has stopped to save
    save: function() {
      // The context here is the wrapped model
      this.mySpecialSaveMethod()
    }
  });
}

export default {
  name: 'setup-ember-autosave',
  initialize: initialize
};
```
